model-optimizer/requirements.txt

```diff
@@ -2,7 +2,7 @@ tensorflow>=1.15.2,<2.0; python_version < "3.8"
 tensorflow>=2.0; python_version >= "3.8"
 mxnet>=1.0.0,<=1.5.1
 networkx>=1.11
-numpy>=1.13.0
+numpy>=1.13.0,<1.19.0
```

Comment:
@lazarevevgeny, does this bug exist for other TF versions or just for 1.15?
may we need to insert a marker for this upper limitation?

Reply:
It is valid for 1.15.X (and we have a lower bound 1.15.2 in the requirements). 
We don't know what will be dependency of 1.16.X, so I would like to avoid setting additional markers for now.